UnoCheck/Solutions/GTK3NotInstalledOnWindowsSolution.cs

```diff
@@ -0,0 +1,52 @@
+﻿using DotNetCheck.Models;
+using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class GTK3NotInstalledOnWindowsSolution : Solution
+	{
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			byte[] data = null;
+
+			WebClient client = new();
+			client.DownloadDataCompleted +=
+			   delegate (object sender, DownloadDataCompletedEventArgs e)
+			   {
+				   data = e.Result;
+			   };
+			
+			client.DownloadDataAsync(new Uri(GtkUrl));
+			while (client.IsBusy)
+			{
+				await Task.Delay(100);
+			}
+
+			string fileName = string.Concat(Guid.NewGuid(), ".exe");
+
+			string fullPath = Path.Combine(Path.GetTempPath(), fileName);
+
+			await File.WriteAllBytesAsync(fullPath, data);
+
+			var gtkResult = ShellProcessRunner.Run(fullPath, "/S");
+
+			if (gtkResult.ExitCode == 0)
+			{
+				ReportStatus($"GTK3+ is {nameof(Status.Ok)}");
+			}
+			else
+			{
+				ReportStatus($"GTK3+ is {nameof(Status.Error)}");
+				throw new InvalidOperationException("GTK3+ cannot be installed.");
+			}
+
+			await Task.CompletedTask;
+		}
+
+		private const string GtkUrl = "https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases/download/2022-01-04/gtk3-runtime-3.24.31-2022-01-04-ts-win64.exe";
```

Comment:
This should come from the manifest file: 

https://github.com/unoplatform/uno.check/blob/1aa1a2c0df19c860b71c68b0d3b356cc21800e15/manifests/uno.ui.manifest.json

Reply:
Done!